### PR TITLE
Adding back urls deleted because of compatibility issues.

### PR DESCRIPTION
**Pull request by ipanova**
GET /pulp/api/v2/consumer_groups/ http://fpaste.org/184316/67088614/
POST /pulp/api/v2/consumer_groups/ http://fpaste.org/184326/67157314/
POST missing required param /pulp/api/v2/consumer_groups/  http://fpaste.org/184318/36710061/
POST invalid param /pulp/api/v2/consumer_groups/  http://fpaste.org/184320/36711661/
GET /pulp/api/v2/content/uploads/ http://fpaste.org/184323/71301142/
POST /pulp/api/v2/content/uploads/ http://fpaste.org/184324/67144414/

### CODE REVIEW

**Comment by ipanova** [comment]
ok test

File: server/pulp/server/webservices/urls.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.conf.urls import patterns, url
 
-from pulp.server.webservices.views.consumer_groups import (ConsumerGroupAssociateActionView,
+from pulp.server.webservices.views.consumer_groups import (ConsumerGroupView,
```
Comment (bmbouter) [inline server/pulp/server/webservices/urls.py]: I usually alphabetize the imports.so `ConsumerGroupView` would come towards the end.

File: server/test/unit/server/webservices/views/test_consumer_groups.py [hunk 2]
```diff
@@ -5,8 +5,9 @@
 
 from base import (assert_auth_CREATE, assert_auth_READ, assert_auth_UPDATE, assert_auth_DELETE,
                   assert_auth_EXECUTE)
-from pulp.server.exceptions import OperationPostponed, MissingResource, InvalidValue
-from pulp.server.webservices.views.consumer_groups import (ConsumerGroupAssociateActionView,
+from pulp.server.exceptions import OperationPostponed, MissingResource, InvalidValue, MissingValue
```
Comment (bmbouter) [inline server/test/unit/server/webservices/views/test_consumer_groups.py]: Please alphabetize

File: server/test/unit/server/webservices/views/test_consumer_groups.py [hunk 3]
```diff
@@ -5,8 +5,9 @@
...
+from pulp.server.webservices.views.consumer_groups import (ConsumerGroupView,
```
Comment (bmbouter) [inline server/test/unit/server/webservices/views/test_consumer_groups.py]: Please alphabetize

File: server/test/unit/server/webservices/views/test_content.py [hunk 4]
```diff
@@ -4,14 +4,14 @@
 
 from django.http import HttpResponseNotFound
 
-from .base import assert_auth_DELETE, assert_auth_READ, assert_auth_UPDATE
+from .base import assert_auth_DELETE, assert_auth_READ, assert_auth_CREATE, assert_auth_UPDATE
```
Comment (bmbouter) [inline server/test/unit/server/webservices/views/test_content.py]: please alphebatize

**Comment by bmbouter** [comment]
@ipanova Please provide stored-forever-fpaste regression testing (webpy vs Django) for both URLs. Demonstrating the error cases with invalid params would also be good.

**Comment by bmbouter** [comment]
@ipanova After regression testing please remove the appropriate web.py code and its associated tests. Now that we're working on master we should remove web.py code with each URL we introduce. Bugfixes will always happen in the right web handler code.

**Comment by bmbouter** [comment]
I added links to this PR in the `URL Conversion` spreadsheet. Once merged please update the spreadsheet to show them as merged.

**Comment by bmbouter** [comment]
flake8 looks good. Test coverage at 100%. Tests pass on all platforms. Nice!

**Comment by bmbouter** [comment]
@ipanova Please fix up with the comments, and let me know when I should re-review.

**Comment by bmbouter** [comment]
@ipanova It looks like the '_href' has an extra slash in it.  http://fpaste.org/184318/36710061/

You can also see the character count differs. Can those be made so they are the same?

**Comment by bmbouter** [comment]
@ipanova Also for http://fpaste.org/184320/36711661/

It seems that u'' is being added to several fields. Also it has that same [extra slash issue as here](https://github.com/pulp/pulp/pull/1627#issuecomment-73947500).

The Content-Length also differs on these. Can this be looked into?

**Comment by bmbouter** [comment]
@ipanova Those two regressions are the only issues. I'm assigning back to you.

**Comment by ipanova** [comment]
@bmbouter  seems like the content length is different because of this extra slash. though not sure why u" is added in some fields - will look into this tomorrow.

**Comment by ipanova** [comment]
right fpaste for https://github.com/pulp/pulp/pull/1627#issuecomment-73947500 http://fpaste.org/184596/37355901/

**Pull request merged**